nbconvert/filters/tests/test_markdown.py

```diff
@@ -137,6 +137,18 @@ def test_markdown2html_math(self):
             "$$a<b&b<lt$$",
             "$$a<b&lt;b>a;a-b<0$$",
             "$$<k'>$$",
+            ("$$x\\n"
```

Comment:
Shouldn't these be actual newlines, rather than making a string with a backslash followed by a lowercase n? You don't type `\n` in markdown, as far as I'm aware. What am I missing?